Packs/CommonScripts/Scripts/CreateIndicatorsFromSTIX/CreateIndicatorsFromSTIX.yml

```diff
@@ -2,19 +2,31 @@ args:
 - description: The entry ID of the STIX file.
   name: entry_id
   required: true
+- auto: PREDEFINED
+  defaultValue: "False"
+  description: Adds Indicators to context.
+  name: context
+  predefined:
+  - "True"
+  - "False"
+- description: Adds tags to Indicators. Comma separated.
+  name: tags
 comment: Creates indicators from the submitted STIX file. Supports STIX 1.0 and STIX 2.x. This automation creates indicators and adds an indicator's relationships if available.
 commonfields:
   id: CreateIndicatorsFromSTIX
   version: -1
+dockerimage: demisto/python3:3.10.12.63474
+enabled: true
+engineinfo: {}
+mainengineinfo: {}
 name: CreateIndicatorsFromSTIX
+pswd: ""
+runas: DBotWeakRole
+runonce: false
 script: '-'
+scripttarget: 0
 subtype: python3
 tags:
 - stix
 - ioc
-timeout: '0'
 type: python
```

Comment:
please remove all the changes from the yml file except:
`dockerimage: demisto/python3:3.10.12.63474`